pkg/addons/data/Deployment.apps/default/docker-registry.yaml

```diff
@@ -15,6 +15,18 @@ spec:
       labels:
         docker-registry: default
     spec:
+      affinity:
+        podAntiAffinity:
```

Comment:
@kwoodson or @kargakis before this merges please could you explain to me in english what the below stanza does?

Reply:
According to the docs, https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature: 
```
The pod anti-affinity rule says that the pod prefers not to be scheduled onto a node if that node is already running a pod with label having key
```

For our infrastructure pods, it will attempt to select another node with the same label.  If it does not find it, it will deploy onto the same node to satisfy the replica count.  If it does find the pod, the it attempts to find another node with the same label to satisfy the anti-affinity and the replica count.

Reply:
@kargakis I thought our scheduler was set up to do this already?  How does adding this change the behaviour?